test/util-test.js

```diff
@@ -434,4 +434,33 @@ describe('util.addFormatMeta()', () => {
       });
     });
   });
+  describe('util.cutAfterJSON()', () => {
+    it('works with simple JSON', () => {
+      assert.equal(util.cutAfterJSON('{"a": 1, "b": 1}'), '{"a": 1, "b": 1}');
+    });
+    it('Cut extra characters after JSON  end', () => {
+      assert.equal(util.cutAfterJSON('{"a": 1, "b": 1}abcd'), '{"a": 1, "b": 1}');
+    });
+    it('tolerant to string constants', () => {
+      assert.equal(util.cutAfterJSON('{"a": "}1", "b": 1}abcd'), '{"a": "}1", "b": 1}');
+    });
+    it('tolerant to string with escaped quoting', () => {
+      assert.equal(util.cutAfterJSON('{"a": "\\\"}1", "b": 1}abcd'), '{"a": "\\\"}1", "b": 1}');
+    });
+    it('works with nested', () => {
+      assert.equal(util.cutAfterJSON(
+        '{"a": "\\\"1", "b": 1, "c": {"test": 1}}abcd'),
+        '{"a": "\\\"1", "b": 1, "c": {"test": 1}}');
+    });
+    it('works with utf', () => {
+      assert.equal(util.cutAfterJSON(
+        '{"a": "\\\"фыва", "b": 1, "c": {"test": 1}}abcd'),
+        '{"a": "\\\"фыва", "b": 1, "c": {"test": 1}}');
+    });
+    it('works with \\\\ in string', () => {
+      assert.equal(util.cutAfterJSON(
+        '{"a": "\\\\фыва", "b": 1, "c": {"test": 1}}abcd'),
+        '{"a": "\\\\фыва", "b": 1, "c": {"test": 1}}');
+    });
+  });
```

Comment:
some newlines to match the rest would be appreciated

Reply:
You mean make arguments on new lines like last 3 tests done ?

Reply:
i know whe're not 100% consistent but sth like a `\n` before the `describe` and some more between those `it`